heareval/predictions/task_predictions.py

```diff
@@ -35,28 +49,76 @@ def __init__(self, nfeatures: int, nlabels: int, prediction_type: str):
         torch.nn.init.normal_(self.projection.weight)
         if prediction_type == "multilabel":
             self.activation: torch.nn.Module = torch.nn.Sigmoid()
+            self.logitloss = torch.nn.BCEWithLogitsLoss()
```

Comment:
I think this should be `self.logit_loss`

Reply:
Done